components/financial_assistance/app/controllers/financial_assistance/applications_controller.rb

```diff
@@ -8,9 +8,11 @@ class ApplicationsController < FinancialAssistance::ApplicationController
     before_action :set_family
     before_action :find_application, :except => [:index, :index_with_filter, :new]
     if EnrollRegistry.feature_enabled?(:bs4_consumer_flow)
-      before_action :enable_bs4_layout, only: [:application_year_selection, :application_checklist, :edit, :eligibility_results, :review_and_submit, :review, :transfer_history,
+      before_action :enable_bs4_layout, only: [:application_year_selection, :application_checklist, :edit, :eligibility_results, :review_and_submit, :review,
                                                :submit_your_application, :wait_for_eligibility_response, :preferences, :application_publish_error, :eligibility_response_error, :index, :index_with_filter]
     end
+    before_action :enable_bs4_layout, only: [:transfer_history] if EnrollRegistry.feature_enabled?(:bs4_admin_flow)
```

Comment:
is this going to work when `bs4_consumer_flow` is on?

my understanding of the ActiveController callbacks like `before_action` is that they may only be set once for any given method. So in this case, if the consumer flag is on, the above before_action will be set, then the admin will get set, which will override the consumer callbck for all of those actions

see my open PR for raw_application for a solution

Reply:
Good catch. Not going to refactor everything right now, but did get things working for this page in particular